### ISSUE TITLE
Average LDDT-Cα and TM-Score on CAMEO

### ISSUE THREAD

**Issue by tyler-hayes** [aqlaboratory/openfold#535]
Hi, thanks for your great work on this open-source implementation of AlphaFold2!

I am trying to reproduce performance on the CAMEO validation set using OpenFold, and I am struggling to do so. I was wondering if you could answer a few questions:
For context, I use `model_1_ptm` with the weights from `openfold/resources/params/params_model_1_ptm.npz`, which were downloaded using the OpenFold installation guide.

1. What is the final average LDDT-Cα on CAMEO for the model? I see the paper says `an OpenFold model trained to completion after the change reached 0.902 lDDT-Cα on our CAMEO validation set, which is almost identical to the score of our prior fully trained model checkpoint.`, so I assume the final performance is around 0.902.
2. How was the LDDT-Cα for these sequences computed? Was the AlphaFol2 implementation of LDDT in JAX used? (https://github.com/google-deepmind/alphafold/blob/020cd6d6cb16540114a084f9dbb8f21f811f9d21/alphafold/model/lddt.py) Or was some other library used?
3. Could you also provide the final average TM-Score on CAMEO for the model?

Thank you in advance for any guidance you can provide! It is greatly appreciated.
